src/Features/Core/Portable/RemoveUnusedParametersAndValues/AbstractRemoveUnusedValuesCodeFixProvider.cs

```diff
@@ -633,6 +640,7 @@ void InsertLocalDeclarationStatement(TLocalDeclarationStatementSyntax declaratio
             bool ShouldRemoveStatement(TLocalDeclarationStatementSyntax localDeclarationStatement, out SeparatedSyntaxList<SyntaxNode> variables)
             {
                 Debug.Assert(removeAssignments);
+                Debug.Assert(localDeclarationStatement != null, "We should never get a null here.");
```

Comment:
```suggestion
                Debug.Assert(localDeclarationStatement != null);
```

IMO, the message is implicit from the condition.

Reply:
I was trying to adhere to the section in the coding conventions [here](https://github.com/dotnet/roslyn/wiki/Contributing-Code#coding-conventions) statong that a message should always be provided. As I am a mew contributer: Is there some guide to follow on when specific rules apply other than common sense?

Reply:
Ah, thanks for following the guidelines so precisely. In general, we avoid the message for null checks, but recommend adding a message where it seems useful. We should likely update the contributor guidelines accordingly.